src/seedu/addressbook/AddressBook.java

```diff
@@ -794,11 +795,11 @@ private static void addPersonToAddressBook(String[] person) {
      * @return true if the given person was found and deleted in the model
      */
     private static boolean deletePersonFromAddressBook(String[] exactPerson) {
-        final boolean changed = ALL_PERSONS.remove(exactPerson);
-        if (changed) {
+        final boolean hasChanged = ALL_PERSONS.remove(exactPerson);
```

Comment:
good job spotting this naming violation. makes it easier to understand that this variable is a boolean/flag.